src/Numerics/Mesh/BrickMesh.jl

```diff
@@ -1126,4 +1126,341 @@ function enumerateboundaryfaces!(elemtoelem, elemtobndy, periodicity, boundary)
     return (bndytoelem, bndytoface)
 end
 
+"""
+    connectmeshfull(comm::MPI.Comm, elemtovert, elemtocoord, elemtobndy,
+                faceconnections)
+
+This function takes in a mesh (as returned for example by `brickmesh`) and
+returns a corner connected mesh.  This returns a `NamedTuple` of:
+
+ - `elems` the range of element indices
+ - `realelems` the range of real (aka nonghost) element indices
+ - `ghostelems` the range of ghost element indices
```

Comment:
do we need the ghost elements because for boundary conditions purposes when using the way that they are applied with DG?

Reply:
Ghost elements are used for communication.